widgets/datechooser/org.eclipse.nebula.widgets.datechooser/src/org/eclipse/nebula/widgets/datechooser/DateChooserObservableValue.java

```diff
@@ -63,7 +63,7 @@ public void widgetDisposed(DisposeEvent e) {
 	};
 
 	public DateChooserObservableValue(final DateChooser dateChooser, int updateEventType) {
-		this(SWTObservables.getRealm(dateChooser.getDisplay()), dateChooser, updateEventType);
+		this(DisplayRealm.getRealm(dateChooser.getDisplay()), dateChooser, updateEventType);
```

Comment:
```suggestion
		this(Realm.getCurrent(), dateChooser, updateEventType);
```
at least that's what is used in the DatabindingFramework internally when no realm is passed.

Reply:
Did you mean Realm.getDefault()?  I don't claim to understand the subtleties of this, so choose to replace it with something that appears to be 100% semantically equivalent with what was there before.  E.g., it looks like getDefault works only on the display thread, but DisplayRealm.getRealm works on any thread, I think.  Of course, I don't know whether that is relevant here either...

Reply:
Yes `Realm.getDefault()` if you search for references you will se that this is always used when no explicit realm is passed in e.g. you can even look at the AbstractObservableValue default constructor that is extended here and I think its better to stick to this convention than trying to be "smart" here, `Realm.getDefault()` works on any thread that is currently a realm thread and one should only create observables from such thread or pass an explicit realm.

Reply:
I see what you mean, e.g., 

![image](https://github.com/eclipse/nebula/assets/208716/4f2e11f2-8eee-4cac-9b39-7a8b476c777b)

Reply:
There was another place using that "pattern" so I changed it too.